### DISCUSSION TITLE
Can Bravo connect to a Fabric Lakehouse\'s Custom Semantic Model?

### DISCUSSION THREAD

**Discussion by rickyschechter** [sql-bi/Bravo#832] [General]
Can Bravo connect to a Fabric Lakehouse\'s Custom Semantic Model (like I can do with Dax Studio)? I am trying to extract it out of the PowerBi report, and I have used it to generate calendar tables, so will like to continue maintaining that. Please advise, thank you!

**Comment by rickyschechter** [comment]
never mind, I found the answer to my own question:

Open Bravo for Power BI.
Navigate to File > Open > Model from DB.
Enter the Power BI connection string in the Server text box. [The connection string typically looks like powerbi://api.powerbi.com/v1.0/myorg/yourworkspace]
Select the desired dataset from the list and click OK.
This should establish the connection to your Fabric semantic model. [If you need to find the connection string, you can locate it in the Power BI service under the settings of your dataset](https://learn.microsoft.com/en-us/fabric/data-warehouse/semantic-models)

but.. the "Manage Dates" feature is not available as it is only supported for models in Power Bi Desktop mode.

**Reply by albertospelta** [reply]
Correct, the Manage Dates feature isn't available when connected to a semantic model in Power BI Service or Fabric; it’s only supported for local models in Power BI Desktop.

**Reply by rickyschechter** [reply]
Thank you for confirming it, Alberto, are you aware if that is a feature that will migrate to those unsupported services in the near future?

**Reply by albertospelta** [reply]
At the moment, there are no plans to extend that feature outside of Power BI Desktop

**Reply by rickyschechter** [reply]
Thank you Alberto!
